SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -1390,9 +1382,10 @@ void SX1276_LoRaRadio::rx_chain_calibration(void)
 
     // Save context
     regPaConfigInitVal = read_register( REG_PACONFIG );
-    initialFreq = (float) (((uint32_t) this->read_register(REG_FRFMSB) << 16) |
+    initialFreq = (uint32_t) ((uint64_t) (((uint32_t) this->read_register(REG_FRFMSB) << 16) |
```

Comment:
Is this more readable as shown below?
```suggestion
    initialFreq = (uint32_t) (
        (uint64_t) (((uint32_t) this->read_register(REG_FRFMSB) << 16) |
        ((uint32_t) this->read_register(REG_FRFMID) << 8) |
        ((uint32_t)this->read_register(REG_FRFLSB))) * (uint64_t) FREQ_STEP
    ) / 100000000;
```

Reply:
I rearranged the statement for readability